internal/evaluator/conftest_evaluator.go

```diff
@@ -622,3 +657,46 @@ func ExtractStringFromMetadata(result output.Result, key string) string {
 	}
 	return ""
 }
+
+func withCapabilities(ctx context.Context, capabilities string) context.Context {
+	return context.WithValue(ctx, capabilitiesKey, capabilities)
+}
+
+// capabilities returns a JSON serialized OPA Capability meant to isolate rego
+// policies from accessing external information, such as hosts or environment
+// variables. If the context already contains the capability, then that is
+// returned as is. Use withCapabilities to pre-populate the context if needed.
+func capabilities(ctx context.Context) (string, error) {
```

Comment:
Would it make sense to rename this to `defaultCapabilities` or maybe `noExternalCapabilities` or even `strictCapabilities`?

Reply:
Good call. Renamed to `strictCapabilities`.